src/shuffle/fwk/gui/ModeIndicator.java

```diff
@@ -106,6 +106,7 @@ private void setup() {
          JLabel label = new JLabel(getString(mode.getI18nKey()));
          label.setFont(modeFont);
          label.setForeground(getColorFor(mode));
+         label.setToolTipText(getString("tooltip."+mode.getI18nKey()));
```

Comment:
For future compatibility for when new entry modes are added, we might want to tweak this slightly.

We can put a check inside the for loop itself to verify that the i18n actually exists:

    String modeTooltipKey = "tooltip." + mode.getI18nKey();
    String modeTooltipText = getString(modeTooltipKey);
    if (!modeTooltipKey.equals(modeTooltipText)) {
        label.setToolTipText(modeTooltipText);
    }

Or, we could either assign it statically below the for loop:

    modeMap.get(EntryMode.PAINT).setToolTipText(getString(KEY_TOOLTIP_PAINT);
    modeMap.get(EntryMode.EXPRESS).setToolTipText(getString(KEY_TOOLTIP_EXPRESS);

From a future-friendly way, the first one might be best (checking if the i18n was retrieved, and if not it won't even set the tooltip text).